app/components/modal/InstructionModal/InstructionModal.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react'
+import Image from 'next/image'
+import {
+  Bs1Circle,
+  Bs2Circle,
+  Bs3Circle,
+  Bs4Circle,
+  Bs5Circle,
+  Bs6Circle,
+} from 'react-icons/bs'
+import styles from './InstructionModal.module.css'
+
+const steps = [
+  {
+    step: 1,
+    title: 'ChatGPT APIキーの取得方法',
+    subtitle: 'Open AI社のアカウントを作成・ログインする',
+    description:
+      'Open AI社のAPIページにアクセスし、「Start building」をクリックしてください。',
+    image: '/SecondModalview.png',
+    alt: 'Open AIのアカウント作成画面',
+    icon: <Bs1Circle className={styles.icon} />, // ステップ1のアイコン
+  },
+  {
+    step: 2,
+    title: 'ChatGPT APIキーの取得方法',
+    subtitle: '支払い情報を登録する-1',
+    description:
+      'TOP画面の左側のネジアイコン「Settings」から「Billing」を選択し、「Add payment details」をクリックします。個人名義で登録する場合は「Individual」、企業名義で登録する場合は「Company」を選択します。',
+    image: '/ThirdModalview.png',
+    alt: '支払い情報登録画面',
+    icon: <Bs2Circle className={styles.icon} />, // ステップ2のアイコン
+  },
+  {
+    step: 3,
+    title: 'ChatGPT APIキーの取得方法',
+    subtitle: '支払い情報を登録する-2',
+    description:
+      'クレジットカードの情報を登録します。支払情報の登録後、「Your subscription was created successfully」と表示されれば、支払い情報の登録は完了です。',
+    image: '/FourthModalview.png',
+    alt: 'クレジットカード',
+    icon: <Bs3Circle className={styles.icon} />, // ステップ3のアイコン
+  },
+  {
+    step: 4,
+    title: 'ChatGPT APIキーの取得方法',
+    subtitle: 'APIキーを取得する-1',
+    description:
+      'TOP画面の左上の鍵アイコン「API keys」を選択し、「+Create new secret key」をクリックします。',
+    image: '/FifthModalview.png',
+    alt: 'API keys-1',
+    icon: <Bs4Circle className={styles.icon} />, // ステップ3のアイコン
+  },
+  {
+    step: 5,
+    title: 'ChatGPT APIキーの取得方法',
+    subtitle: 'APIキーを取得する-2',
+    description:
+      'NameでAPIキーの名前を決め、「Create secret key」をクリックします。',
+    image: '/SixthModalview.png',
+    alt: 'API keys-2',
+    icon: <Bs5Circle className={styles.icon} />, // ステップ3のアイコン
+  },
+  {
+    step: 6,
+    title: 'ChatGPT APIキーの取得方法',
+    subtitle: 'APIキーを取得する-3',
+    description:
+      'ポップアップ表示される「Create new secret key」から、APIキーの情報をコピーしてメモしましょう。',
+    image: '/SeventhModalview.png',
+    alt: 'API keys-3',
+    icon: <Bs6Circle className={styles.icon} />, // ステップ3のアイコン
+  },
+  // さらに必要なステップを追加
+]
+
+type InstructionModalProps = {
+  onClose: () => void
+  onFirst: () => void
+}
+
+const InstructionModal: React.FC<InstructionModalProps> = ({
+  onClose,
+  onFirst,
+}) => {
+  const [currentStep, setCurrentStep] = useState(0)
+
+  const handleNext = () => {
+    setCurrentStep((prev) => (prev < steps.length - 1 ? prev + 1 : prev))
+  }
+
+  const handleBack = () => {
+    setCurrentStep((prev) => (prev > 0 ? prev - 1 : prev))
+  }
+
+  return (
+    <div className={styles.modalOverlay}>
+      <div className={styles.modalContent}>
+        <button className={styles.closeButton} onClick={onClose}>
+          &times;
```

Comment:
React Iconsにちょうどよいのがあります！
他のモーダルでも直せるとよいですね。
```suggestion
          <RxCross1 />
```